frontend/src/metabase/home/components/HomeContent/HomeContent.tsx

```diff
@@ -53,7 +52,18 @@ export const HomeContent = (): JSX.Element | null => {
   }
 
   if (isXraySection(databases, isXrayEnabled)) {
-    return <HomeXraySection />;
+    return (
+      <>
+        <HomeXraySection />
+        {isAdmin && showEmbedHomepage && (
```

Comment:
Yeah I think that's an unlikely case as this card only shows up the first time the installer opens metabase, but it could happen so I added it [here](https://github.com/metabase/metabase/pull/40168/commits/901ef05ebd40d1408624bdc4cb67fabc5b0bdb09#diff-02d10a444f09b42d81fb791fa97d646113794fab34cbeac005907d1c15a119c0L69-L71).
The code is a bit repetitive but I'll get removed soon with the full version